resources/sass/components/_list_pages.scss

```diff
@@ -1,39 +1,22 @@
-.oe-list-pages-sort-form {
-  display: flex;
-  justify-content: flex-end;
-  align-content: center;
-}
-
 .oe-list-pages-sort-form .form-item {
   margin: 0 0 0 1.5rem;
-}
-.oe-list-pages-sort-form .form-label,
-.oe-list-pages-sort-form .form-select {
-  display: inline-block;
-  width: auto;
-}
-
-.oe-list-pages-sort-form .form-label {
-  margin-right: 1.5rem;
-}
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
 
-.oe-list-pages-sort-form .form-select {
-  @media (min-width: 576px) {
-    max-width: calc(100% - 75px);
+  .form-label {
```

Comment:
dont think this is ok, and its worse for me now, see
![Screenshot 2024-06-05 at 18 15 43](https://github.com/openeuropa/oe_whitelabel/assets/11507647/1f7687e8-f569-4dda-b4a5-e2adbb824f09)